src/FixerDefinition/FixerDefinition.php

```diff
@@ -36,9 +36,9 @@ public function __construct(
         $summary,
         $description,
         array $codeSamples,
-        $configurationDescription,
+        $configurationDescription = null,
         array $defaultConfiguration = null,
-        $riskyDescription
+        $riskyDescription = null
```

Comment:
it gets kind of bloated to add some dummy data for each fixer that is not risky.